### DISCUSSION TITLE
Codespaces will not load

### DISCUSSION THREAD

**Discussion by SilverSlime0** [community/community#86529] [Codespaces]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/150315493/087e63fa-75b3-472f-9ac8-707922b06198)
Github codespace will not load, this problem persists with new codespaces generated and across browsers and devices. I have already attempted accessing from [github.com/codespaces](https://github.com/codespaces)

**Comment by velazcopolo** [comment]
i have the same problem,
